data/json/items/comestibles/brewing.json

```diff
@@ -0,0 +1,272 @@
+[
```

Comment:
I'll publish the `jq` script that sets the field ordering as it makes most sense to perform that change when migrating items